tests/unit/lib/DynamicArray_unittest.cc

```diff
@@ -251,10 +254,35 @@ TEST_F(CDynamicArrayFixture, append_array_bool)
 	EXPECT_EQ(wrapper_array_b->get_element(2), true);
 }
 
-TEST_F(CDynamicArrayFixture, serializable_array)
+TYPED_TEST(CDynamicArrayTest, save_serializable)
 {
-	wrapper_array_i->save_serializable_pre();
-	wrapper_array_i->load_serializable_pre();
-	wrapper_array_b->save_serializable_pre();
-	wrapper_array_b->load_serializable_pre();
+	for (int32_t i = 0; i < 5; i++)
+	{
+		this->custom_array->push_back((TypeParam)CMath::random(0, 1));
+	}
+
+	/* generate file name */
+	char filename[] = "serialization-asciiCDynamicArray.XXXXXX";
```

Comment:
mmm, I get Bus error if I change it with  `char* filename = (char*)"serialization-asciiCDynamicArray.XXXXXX";` And it will happened in `mkstemp`. So I guess it's because I haven't assign a length of memory to that pointer. Anyway, I think we don't need to worry about it should be a pointer or not :)